src/rpc/client.cpp

```diff
@@ -315,91 +368,140 @@ static const CRPCConvertParam vRPCConvertParams[] =
     { "stop", 0, "wait" },
     { "addnode", 2, "v2transport" },
     { "addconnection", 2, "v2transport" },
+    { "decodepsbt", 0, "psbt", ParamFormat::STRING },
+    { "analyzepsbt", 0, "psbt", ParamFormat::STRING},
+    { "verifymessage", 1, "signature", ParamFormat::STRING },
+    { "verifymessage", 2, "message", ParamFormat::STRING },
+    { "getnewaddress", 0, "label", ParamFormat::STRING },
+    { "getnewaddress", 1, "address_type", ParamFormat::STRING },
+    { "backupwallet", 0, "destination", ParamFormat::STRING },
+    { "echoipc", 0, "arg", ParamFormat::STRING },
+    { "encryptwallet", 0, "passphrase", ParamFormat::STRING },
+    { "getaddressesbylabel", 0, "label", ParamFormat::STRING },
+    { "loadtxoutset", 0, "path", ParamFormat::STRING },
+    { "migratewallet", 0, "wallet_name", ParamFormat::STRING },
+    { "migratewallet", 1, "passphrase", ParamFormat::STRING },
+    { "setlabel", 1, "label", ParamFormat::STRING },
+    { "signmessage", 1, "message", ParamFormat::STRING },
+    { "signmessagewithprivkey", 1, "message", ParamFormat::STRING },
+    { "walletpassphrasechange", 0, "oldpassphrase", ParamFormat::STRING },
+    { "walletpassphrasechange", 1, "newpassphrase", ParamFormat::STRING },
```

Comment:
listlabels and setwalletflag should be added?
I assume these are not included because these params are string but only accept pre-defined string.
IMHO, adding these params to the list and generate consistent error message seems good.

In the case of listlabels, when "=" including params are passed with -name:
```
$ build/bin/bitcoin-cli -regtest -named listlabels invalid=purpose
error code: -8
error message:
Unknown named parameter invalid
```
Without -named:
```
build/bin/bitcoin-cli -regtest listlabels invalid=purpose
error code: -8
error message:
Invalid 'purpose' argument, must be a known purpose string, typically 'send', or 'receive'.
```

By adding listlabels to the list, the error will be consistent with both cased.

Reply:
>